foundry.toml

```diff
@@ -2,5 +2,5 @@
 src = 'src'
 out = 'out'
 libs = ['lib']
-
+ffi=true
```

Comment:
Let's not do that on the config level, better pass `--ffi` explicitly when we need it.
Usually I'd be fine with `ffi=true` but it's a community project, let's err on the safer side.